acceptance/acceptance_test.go

```diff
@@ -484,6 +485,32 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 	}
 }
 
+func hasKey(list []string, key string) bool {
+	for _, keyvalue := range list {
+		items := strings.SplitN(keyvalue, "=", 2)
+		if len(items) == 2 && items[0] == key {
+			return true
+		}
+	}
+	return false
+}
+
+func addEnvVar(t *testing.T, env []string, repls *testdiff.ReplacementsContext, key, value string) []string {
```

Comment:
nit: 90% of this method deals with replacements rather than adding an a variable to an environment, could there be a more descriptive name for this method to reflect that?

Reply:
It's a very local helper, not exported function, but we can change the name if you have good ideas?